src/vector/mod.rs

```diff
@@ -1,20 +1,25 @@
-use num_traits::Zero;
-use std::fmt::{Debug, Display};
-use std::ops::{Add, AddAssign, Div, DivAssign, Index, IndexMut, Mul, MulAssign, Sub, SubAssign};
-
+use crate::scalar::Scale;
 use crate::{IndexType, Scalar};
+use num_traits::Zero;
+use std::fmt::Debug;
+use std::ops::{Add, AddAssign, Div, Index, IndexMut, Mul, MulAssign, Sub, SubAssign};
 
-mod serial;
+// #[cfg(feature = "faer")]
+mod faer_serial;
+#[cfg(feature = "nalgebra")]
+mod nalgebra_serial;
 
-pub trait VectorIndex: Sized + Index<IndexType, Output = IndexType> + Debug + Display {
+pub trait VectorIndex: Sized + Index<IndexType, Output = IndexType> + Debug {
+    //+ Display
```

Comment:
if this is not needed then I'm happy to remove

Reply:
✅